data/data/aws/variables-aws.tf

```diff
@@ -17,10 +17,14 @@ variable "aws_master_instance_type" {
   description = "Instance type for the master node(s). Example: `m4.large`."
 }
 
-variable "aws_ec2_ami_override" {
+variable "aws_ami" {
   type        = "string"
-  description = "(optional) AMI override for all nodes. Example: `ami-foobar123`."
-  default     = ""
+  description = "AMI for all nodes.  An encrypted copy of this AMI will be used.  Example: `ami-foobar123`."
+}
+
+variable "aws_ami_region" {
```

Comment:
In the current setup, we should be using the ami from the current region only. Why is this new variable requir d.

Reply:
> In the current setup, we should be using the ami from the current region only.

In my testing, I saw no difference in the time required to copy from us-east-1->us-east-2 and us-east-1->ap-southeast-2 (~5mins).  I'll try us-east-1->us-east-1 today, but if it turns out to also be ~5min, this variable sets the stage for RHCOS not being published to all regions.

Reply:
Even is it possible, I'm concerned that we add this capability to installer terraform that is not yet required and we know is not solution we want to use long term.


And if this is not required for the purpose of this PR, I would be more inclined towards moving this discussion to another PR / issue

Reply:
> And if this is not required for the purpose of this PR, I would be more inclined towards moving this discussion to another PR / issue

Ok.  Punted with f08e266 -> 4fcef5e.